synapse/storage/prepare_database.py

```diff
@@ -25,7 +25,7 @@
 
 # Remember to update this number every time a change is made to database
 # schema files, so the users will be informed on server restarts.
-SCHEMA_VERSION = 51
+SCHEMA_VERSION = 52
```

Comment:
someone (@erikjohnston ?) remind me what our policy is for when we have to bump this? is it written down somewhere?

Reply:
My reason for bumping it here is that the original e2e room keys tables were added in 51 and this changes obviously needs to happen after that.

Reply:
well, that seems a good reason. Another good reason is that we are already up to 52 in develop; I'm not sure why it is being shown as a change here.